lib/Literal/DateTime.php

```diff
@@ -66,8 +66,8 @@ public function __construct($value = null, $lang = null, $datatype = null)
         }
 
         // Convert DateTime objects into string
-        if ($value instanceof \DateTime) {
-            $atom = $value->format(\DateTime::ATOM);
+        if ($value instanceof \DateTimeInterface) {
+            $atom = $value->format(\DateTimeInterface::ATOM);
```

Comment:
It looks like the `\DateTimeInterface::ATOM` constant was added in PHP 7.2.0:
https://www.php.net/manual/en/class.datetime.php#datetime.changelog

Is there any issue with using `\DateTime::ATOM` instead, in order to support PHP 7.1?

Reply:
Yes, as the constants are inherited by DateTime from DateTimeInterface.